app/views/shared/tags/_curated_links_preview.html.erb

```diff
@@ -26,6 +34,14 @@
 <h4>Uncurated items <em>(not shown to user)</em></h4>
 <ul>
 <% tag.uncurated_tagged_documents.each do |item| %>
-  <li><%= item.title %></li>
+  <li class="tagged-document">
+    <% if item.base_path %>
+      <a href=<%= Plek.new.find('content-tagger') + '/taggings/lookup' + item.base_path %>>
```

Comment:
Could you do the `Plek.new.find` in the controller and then pass it into the view? It feels more like business logic than presentation logic.